design/baremetal-operator/bmh_non-bootable_iso.md

```diff
@@ -0,0 +1,138 @@
+# Add non-bootable iso attach API to BareMetalHost
+
+## Status
+
+Implementable
+
+## Summary
+
+Add a new Custom Resource(CR), `DataImage`, to the cluster which will make it possible,
+using ownerReferences (example : [HostFirmwareSettings](https://github.com/metal3-io/baremetal-operator/blob/9ce684e6462a6ab55ff650cb6c11f9ba1ffb395d/docs/api.md?plain=1#L664)) and same `Name` and `Namespace` as the `BareMetalHost`, to attach a generic non-bootable ISO (a CD "ISO 9660" image) image via Ironic, after the host has been provisioned.
```

Comment:
I'd suggest moving ownerReferences and the example to the implementation details: it's too detailed for the summary.

Reply:
Done.